books.js

```diff
@@ -1,61 +1,71 @@
-let removeButton = [];
 const titleInput = document.getElementById('title');
 const authorInput = document.getElementById('author');
-let booksList = [];
 const bookContainer = document.getElementById('books-container');
+const addButton = document.querySelector('.form-container .add-btn');
 
-function createHtmlDiv(listOfBooks) {
-  let books = '';
-  listOfBooks.forEach((div) => {
-    books += `<article class="author-title-div">
-    <h3>${div.authorName}</h3>
-    <h3>${div.titleName}</h3>
-    <button class="remove-btn">Remove</button>
-    <hr>
-    </article>`;
-  });
-  bookContainer.innerHTML = books;
-  removeButton = document.querySelectorAll('button.remove-btn');
-  removeButton.forEach((btn, index) => {
-    btn.addEventListener('click', (e) => {
-      e.preventDefault();
-      function removeBookFromCollection(id) {
-        booksList = JSON.parse(localStorage.getItem('storageFormData'));
-        const filteredArray = booksList.filter((book) => booksList.indexOf(book) !== id);
-        localStorage.setItem('storageFormData', JSON.stringify(filteredArray));
-        createHtmlDiv(filteredArray);
-      }
-      removeBookFromCollection(index);
+class CreateBook {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.removeButton = [];
+    this.booksList = [];
```

Comment:
- [ ] [Optional] I would suggest you use unique id property for each book since it would make it easier to remove books with duplicate values.